helix-core/src/main/java/org/apache/helix/manager/zk/CallbackHandler.java

```diff
@@ -131,6 +136,10 @@
   private boolean _batchModeEnabled = false;
   private boolean _preFetchEnabled = true;
   private HelixCallbackMonitor _monitor;
+  private final long _periodicRefreshInterval;
```

Comment:
For generosity, shall we call it periodicCallbackInterval, or periodicTriggerInterval?